reference/idp-support/azure.md

```diff
@@ -16,13 +16,13 @@ To use Azure as your IdP, you will first need to register an OAuth application w
         
         **Note:** Azure AD requires the redirect endpoint to handle TLS
       - Under `Advanced settings`, make sure the application is issuing `Access tokens` by checking next to the box that says `Access tokens`
-      - Under `Supported account types` select whichever option fits your usecase
+      - Under `Supported account types` select whichever option fits your use case
 
 5. Click on `Certificates & secrets` in the left sidebar. Click `+ New client secret` and set the expiration date you wish. Copy the value of this secret somewhere. You will need it when configuring Ambassador Edge Stack.
 
 ## Set up Ambassador Edge Stack
 
-After configuring an OAuth application in Azure AD, configuring Ambassador Edge Stack to make use it for authentication is simple.
+After configuring an OAuth application in Azure AD, configuring Ambassador Edge Stack to make use for authentication is simple.
```

Comment:
```suggestion
After configuring an OAuth application in Azure AD, configuring Ambassador Edge Stack to make use of it for authentication is simple.
```
I think the original problem was a missing "of"; your version didn't read right either.